### DISCUSSION TITLE
那个V2是什么版本？

### DISCUSSION THREAD

**Discussion by thedespair** [cangzhang/ChampR#241] [Q&A]
刚才我的1.3.11不能用了，但是我看到了V2版本和V1版本，我想问这个是什么？

我开启的时候会调用Edge浏览器，导致我的游戏会卡顿掉帧
win11 22H2

**Comment by MSamor** [comment]
我也是，发现1.3.11不能用了，然后下了2.0，然后就能用了，感觉两个版本调的接口不一样。安装包小了好多，很舒服。

**Reply by thedespair** [reply]
因为我英文学的不太好，现在我不知道该怎么去使用他。可以告诉我步骤吗？

**Reply by MSamor** [reply]
> 因为我英文学的不太好，现在我不知道该怎么去使用他。可以告诉我步骤吗？

不用绑定安装目录了，先打开作者的软件，然后直接打开LOL客户端就行了

**Reply by thedespair** [reply]
他开启时会调用Edge浏览器，一直开这会非常卡，还掉帧

**Comment by wesleyel** [comment]
使用[tauri](https://github.com/tauri-apps/tauri)写的船新后端呢，学习一个😋
